tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "gts/tslint.json"
+  "extends": "gts/tslint.json",
+  "rules": {
```

Comment:
these new rules were added during @alexander-fenster's generation of the client, and I believe are automatic.

Reply:
We'll have two instances of `@ts-ignore` for now (in the place where we load protos). They might go away later if I figure out how to deal with that. Those are only for dealing with dynamic protos.